docs/podman-rm.1.md

```diff
@@ -17,7 +17,9 @@ Remove all containers.  Can be used in conjunction with -f as well.
 
 **--force, f**
 
-Force the removal of a running and paused containers
+Force the removal of a running and paused containers.  Forcing a container removal will also
+remove containers from container storage, if the container is not known to podman.  The container
+could have been created by a different container engine. 
```

Comment:
I'm hemming and hawing about whether or not to add something like: "different container engine and due to that this option should be used cautiously."  IDK if that's beating over the head too much or not.

Reply:
Yes I was going down that path. I also was thinking that this can happen when you are using just podman in some weird situations.  Where the database gets updated and for some reason storage does not get removed.